### PR TITLE
cb166 Credit Product component

### PR DESCRIPTION
**Pull request by novadeveloper14**
image

### CODE REVIEW

File: src/components/cards/CreditProductCard/styles.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  overflow: hidden;
+  width: 217px;
+  height: 414px;
+  border-radius: 8px;
+  outline: 2px solid
+    ${({ theme }) => theme?.palette?.neutral?.N30 || inube.palette.neutral.N30};
+  background-color: ${({ theme }) =>
+    theme?.palette?.neutral?.N0 || inube.palette.neutral.N0};
+  box-shadow: 0px 1px 3px 0px rgba(9, 30, 66, 0.13);
+  box-shadow: 0px 4px 8px 3px rgba(9, 30, 66, 0.13);
```
Comment (diegoAndres01) [inline src/components/cards/CreditProductCard/styles.ts:16]: The box-shadow property is repeated

**Review by diegoAndres01 — CHANGES_REQUESTED** [review]

File: src/components/cards/CreditProductCard/styles.ts (hunk 1 above)
Comment (novadeveloper14) [reply src/components/cards/CreditProductCard/styles.ts:15]: The properties were not the same, it was copied from what was shown in figma, anyway the box-shadow was removed which did not represent a big change in the appearance of the component

**Review by diegoAndres01 — APPROVED** [review]

File: src/components/cards/CreditProductCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  termMonths: number;
+  periodicFee: number;
+  paymentCycle: string;
+  onEdit: () => void;
+  onDelete: () => void;
+}
+
+function CreditProductCard(props: CreditProductCardProps) {
+  const {
+    title,
```
Comment (carin91) [inline src/components/cards/CreditProductCard/index.tsx:17]: Change the name of this prop to "Line of Credit"

File: src/components/cards/CreditProductCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    paymentMethod,
+    loanAmount,
+    interestRate,
+    termMonths,
+    periodicFee,
+    paymentCycle,
```
Comment (carin91) [inline src/components/cards/CreditProductCard/index.tsx:23]: Change the name of this prop to "Schedule" + add the interface based on the next enumerator: 
weekly, 10-day intervals, biweekly, semimonthly, monthly, bimonthly, quarterly, semiannually, annually.

File: src/components/cards/CreditProductCard/interface.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  return (
+    <StyledCreditProductCard>
+      <Stack direction="column" height="100%" padding="12px" gap="8px">
+        <Text size="large" appearance="gray" weight="bold" margin="0px 0px 8px">
+          {capitalizeFirstLetter(truncateTextToMaxLength(title, 22))}
+        </Text>
+
+        <Stack direction="column">
+          <Text size="small" appearance="gray" weight="bold">
+            Medio de pago
```
Comment (carin91) [inline src/components/cards/CreditProductCard/interface.tsx:37]: Avoid magical words, relocate this text in a config file

File: src/components/cards/CreditProductCard/interface.tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+          </Text>
+          <Text>
+            {capitalizeFirstLetter(truncateTextToMaxLength(paymentMethod, 40))}
+          </Text>
+        </Stack>
+        <Stack direction="column">
+          <Text size="small" appearance="gray" weight="bold">
+            Monto del prestamo
```
Comment (carin91) [inline src/components/cards/CreditProductCard/interface.tsx:45]: Avoid magical words, relocate this text in a config file

File: src/components/cards/CreditProductCard/interface.tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+          </Text>
+          <Text>{loanAmount === 0 ? "$ 0" : currencyFormat(loanAmount)}</Text>
+        </Stack>
+        <Stack direction="column">
+          <Text size="small" appearance="gray" weight="bold">
+            Tasa de interés
```
Comment (carin91) [inline src/components/cards/CreditProductCard/interface.tsx:51]: Avoid magical words, relocate this text in a config file

File: src/components/cards/CreditProductCard/interface.tsx [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+          </Text>
+          <Text>{interestRate} %</Text>
+        </Stack>
+        <Stack direction="column">
+          <Text size="small" appearance="gray" weight="bold">
+            Plazo en meses
```
Comment (carin91) [inline src/components/cards/CreditProductCard/interface.tsx:57]: Avoid magical words, relocate this text in a config file

File: src/components/cards/CreditProductCard/interface.tsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+          </Text>
+          <Text>{termMonths}</Text>
+        </Stack>
+        <Stack direction="column">
+          <Text size="small" appearance="gray" weight="bold">
+            Cuota periodica K+I
```
Comment (carin91) [inline src/components/cards/CreditProductCard/interface.tsx:63]: Avoid magical words, relocate this text in a config file

File: src/components/cards/CreditProductCard/interface.tsx [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+          </Text>
+          <Text>{periodicFee === 0 ? "$ 0" : currencyFormat(periodicFee)}</Text>
+        </Stack>
+        <Stack direction="column">
+          <Text size="small" appearance="gray" weight="bold">
+            Ciclo de pagos
```
Comment (carin91) [inline src/components/cards/CreditProductCard/interface.tsx:69]: Avoid magical words, relocate this text in a config file

**Review by carin91 — APPROVED** [review]

**Pull request merged**